src/backend/marsha/core/views.py

```diff
@@ -83,9 +82,10 @@ def get_context_data(self):
                 "state": "error",
                 "resource": self.model._meta.verbose_name_plural.lower(),
                 "resource_data": "null",
+                "editable": False,
             }
 
-        context = {"state": STUDENT if lti.is_student else INSTRUCTOR}
+        context = {"state": "success"}
```

Comment:
do we really still need the state then?

Reply:
we can maybe determine the app is in error if there is no jwt token in the lti view.

Reply:
So yes we can rely on the presence or not of the jwt token and in fact the application does not work without it (it's a bug)

Reply:
I thought it was always "success" . It makes sense to keep it if it can also be "error".